bagel/cli.py

```diff
@@ -132,10 +132,12 @@ def pheno(
             _assessments = [
                 models.Assessment(identifier=tool)
                 for tool, columns in tool_mapping.items()
-                if putil.are_not_missing(columns, _sub_pheno, data_dictionary)
+                if putil.are_any_nonmissing(
```

Comment:
:cherries:: I think double-negatives are hard to think about (I know that's what we already had). How about

```suggestion
                if putil.are_any_available(
```